controls/V-72047.rb

```diff
@@ -20,6 +20,12 @@
 -----------------
 =end
 
+ww_dir_groups = attribute(
+  'ww_dir_groups',
+  default: ['root', 'sys', 'bin'],
+  description: "Groups world-writeable directories are allowed to be owned by"
```

Comment:
This doesn't match the policy prose.

Application groups are < 500 on EL6 and < 1000 on EL7 systems by default.

Also, this should not be global.